Packs/CertStream/Playbooks/playbook-Create_list_for_PTH.yml

```diff
@@ -668,43 +353,635 @@ tasks:
+          "x": 340,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Please provide the primary domain name?
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: exampleDomain
+        tooltip: The domain name for which to hunt homographs list. e.g. exampleDomain
+        readonly: false
+      - id: "1"
+        label: ""
+        labelarg:
+          simple: Please provide the list of homographs to match?
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: 'Note: If using LLM, this field should be empty. The list of primary
+          domain''s homographs to match. e.g. ```examp1e, ex4mple, exampl3, xample```'
+        readonly: false
+      title: Add a single domain to hunt
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: c9a81353-1447-494d-8e8b-2c2701da63e0
+    type: collection
+    task:
+      id: c9a81353-1447-494d-8e8b-2c2701da63e0
+      version: -1
+      name: Add multiple domains to hunt
+      description: ""
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "35"
+    scriptarguments:
+      extend-context:
+        simple: ListData=.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -90,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Add your website manually - in JSON format (see example above)
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: 'List of homographs to hunt. provided in the following format:
```

Comment:
Please change the 'permu7ation1' ..... to 'domain1_homograph1' as it appears in the short description.